app/index.deck

```diff
@@ -175,11 +175,14 @@
   </script>
 
   <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
+    window.addEventListener('load', () => {
+      if (window.spinnakerSettings && window.spinnakerSettings.analytics.ga) {
```

Comment:
do we support any other fields in `analytics` that are used without `ga`?

Reply:
yeah, good catch